ruby_programming/intermediate_ruby/oop_project.md

```diff
@@ -224,6 +224,7 @@ Build a Mastermind game from the command line where you have 12 turns to guess t
 Send us your solution so we can show others! Submit a link to the Github repo with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).  Please include your partner's github handle somewhere in the description if they would like attribution.
 
 * Add your solution below this line!
+* [OthmanAmoudi's solution] (https://github.com/OthmanAmoudi/Mastermind-Game-CLI) - [View in Browser - fancy cli] (https://repl.it/@theweeknd/Mastermind-You-are-the-CodeBreaker)
```

Comment:
Please remove the spaces between the closing braces and opening parenthesis for the links. `[display text](url address)`